From c08b809d8c2cd32ce14715f6c5712dbcdd920b45 Mon Sep 17 00:00:00 2001
From: Hans de Goede <johannes.goede@oss.qualcomm.com>
Date: Tue, 28 Jul 2026 11:26:09 +0200
Subject: [PATCH 07824/15333] drm/msm: Fix stale comments in uapi/drm/msm_drm.h

At some point drm_msm_gem_syncobj was renamed to drm_msm_syncobj and
MSM_SUBMIT_SYNCOBJ_FLAGS was renamed to MSM_SYNCOBJ_FLAGS but some
comments still refer to the old names.

Update the comments with the new names.

Signed-off-by: Hans de Goede <johannes.goede@oss.qualcomm.com>
Patchwork: https://patchwork.freedesktop.org/patch/742717/
Message-ID: <20260728092609.22049-1-johannes.goede@oss.qualcomm.com>
Signed-off-by: Rob Clark <robin.clark@oss.qualcomm.com>
---
 include/uapi/drm/msm_drm.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/uapi/drm/msm_drm.h b/include/uapi/drm/msm_drm.h
index 7f2e594be4eb..3d363f12f66a 100644
--- a/include/uapi/drm/msm_drm.h
+++ b/include/uapi/drm/msm_drm.h
@@ -228,7 +228,7 @@ struct drm_msm_gem_cpu_fini {
 
 struct drm_msm_syncobj {
 	__u32 handle;     /* in, syncobj handle. */
-	__u32 flags;      /* in, from MSM_SUBMIT_SYNCOBJ_FLAGS */
+	__u32 flags;      /* in, from MSM_SYNCOBJ_FLAGS */
 	__u64 point;      /* in, timepoint for timeline syncobjs. */
 };
 
@@ -393,9 +393,9 @@ struct drm_msm_vm_bind {
 	__s32 fence_fd;
 	/** @queue_id: in, submitqueue id */
 	__u32 queue_id;
-	/** @in_syncobjs: in, ptr to array of drm_msm_gem_syncobj */
+	/** @in_syncobjs: in, ptr to array of drm_msm_syncobj */
 	__u64 in_syncobjs;
-	/** @out_syncobjs: in, ptr to array of drm_msm_gem_syncobj */
+	/** @out_syncobjs: in, ptr to array of drm_msm_syncobj */
 	__u64 out_syncobjs;
 	/** @nr_in_syncobjs: in, number of entries in in_syncobj */
 	__u32 nr_in_syncobjs;
-- 
2.55.0

